charts/operator/templates/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
         - --configurator-tag={{ include "configurator.tag" . }}
-        - --configurator-base-image={{ .Values.configurator.repository }}
+        - --configurator-base-image={{ .Values.configurator.containerRegistry }}
```

Comment:
why this change?

Reply:
I was wondering that, too.

Reply:
I highly recommend reading my commit message. I will put the whole reasoning into PR cover letter

```
operator: Change configurator container registry values yaml
Currently nightly build is failing with the following error:
```
2023/09/15 01:26:17 Error querying repository tags: unable to open repository 'docker.redpanda.com/redpandadata/redpanda-operator
docker.redpanda.com/redpandadata/configurator': invalid reference: invalid repository
```

The values.yaml file for operator has 2 repository values which confuses `bump_chart_version.sh` script.

The `docker-tag-list` program receives in the REPO variable the following content:
```
docker.redpanda.com/redpandadata/redpanda-operator
docker.redpanda.com/redpandadata/configurator
```

This change overcome this issue.

P.S. New version of the Operator will not use configurator container.

REF
https://github.com/redpanda-data/helm-charts/actions/runs/6192749166/job/16813278284#step:6:7
```